pwndbg/commands/heap.py

```diff
@@ -39,25 +39,34 @@ def format_bin(bins, verbose=False, offset=None):
     result = []
     for size in bins:
         b = bins[size]
-        if isinstance(b, tuple):
-            chain, count = b
-        else:
-            chain = b
-            count = None
 
-        if not verbose and (chain == [0] and not count):
+        count, is_chain_corrupted = None, False
+        if len(b) == 1:  # fastbin:
+            chain_fd = b
+        elif len(b) == 2:  # tcachebin:
```

Comment:
What is this dictated by? 

Can we add a comment with link to some documentation or glibc source code? Or explain this?